include/cpprealm/networking/http.hpp

```diff
@@ -105,8 +105,8 @@ namespace realm::networking {
     // Interface for providing http transport
     struct http_transport_client {
         virtual ~http_transport_client() = default;
-        virtual void send_request_to_server(const request& request,
-                                            std::function<void(const response&)>&& completion) = 0;
+        virtual void send_request_to_server(::realm::networking::request &&request,
+                                            std::function<void(const response &)> &&completion) = 0;
```

Comment:
`git clang-format` formatted these files differently than what is currently configured for realm-core. The `.clang-format` says it was based off the CLion configuration.